package.json

```diff
@@ -55,6 +55,7 @@
 		"re-resizable": "4.7.1",
 		"react": "16.4.1",
 		"react-dom": "16.4.1",
+		"react-sortable-hoc": "0.8.3",
```

Comment:
How many bytes does this add to our bundle size?

Reply:
Around 40 KB